src/conn/conn_cache.c

```diff
@@ -8,6 +8,32 @@
 
 #include "wt_internal.h"
 
+/*
+ * __cache_config_abs_to_pct --
+ *	Convert the absolute value in cache config to percentage.
+ */
+static inline int
+__cache_config_abs_to_pct(WT_SESSION_IMPL *session,
+    double *param, const char *param_name)
+{
+	WT_CONNECTION_IMPL *conn;
+
+	conn = S2C(session);
+
+	/*
+	 * Anything above 100 is an absolute value; convert it to percentage.
+	 * An absolute value should not exceed the cache size.
+	 */
+	if (*param > 100.0) {
+		if (*param > conn->cache_size)
+			WT_RET_MSG(session, EINVAL,
+			    "%s should be lower than cache size!", param_name);
```

Comment:
We don't use exclamation mark in any other message. Can drop it, I do not feel this one strongly.